front/app/components/FormBuilder/utils.tsx

```diff
@@ -59,6 +59,9 @@ export type FormBuilderConfig = {
   showStatusBadge: boolean;
   isLogicEnabled: boolean;
   alwaysShowCustomFields: boolean;
+  /* For Ideation, when you configure the form it gets applied to ALL ideation phases within a project, 
```

Comment:
Realized it's also true when the participation method is `voting`. Will adjust in a different PR. cc @amanda-anderson